### ISSUE TITLE
Can it be that your WIN-Installers got MALWARE- please react and give explanation

### ISSUE THREAD

**Issue by zumbatan** [paritytech/parity#7726]
Hi there,
i have read about some current problems concerning your installers. But they were related to the parity.io website.

In our case this very GITHUB page https://github.com/paritytech/parity/releases  seems to be infected?!
Precisely, we are talking about the **1.7.13 stable  file  x86_64 	InstallParity.exe**   (checksum matched after download)

Can you guys please profoundly check whether our suspicion is valid or not?
Concretely, we got the malware error regarding the "UNINSTALL.EXE" File which is being created during the installation process.  Precise detection name was:  "QVM20.1.B8E1.Malware.Gen"

Hope you reply soon , hundreds of people are downloading your tools.
If from your expert view, this really should be some malware, please tell us how to proceed professionally to be 100% secure from it.

Thank you very much

@5chdn   any official explanations please?? we have to know which files are 100% SECURE

**Comment by jfnz** [comment]
Yeah, what's going on? When I downloaded on Monday I was given this file:

![image](https://user-images.githubusercontent.com/12504630/35592686-a6c61692-0672-11e8-90d9-68a8b8a154bb.png)

![image](https://user-images.githubusercontent.com/12504630/35592751-db7301a2-0672-11e8-92d8-4b53b4c7926c.png)

Then I came back a day later and was given this file:

![image](https://user-images.githubusercontent.com/12504630/35592666-9b105c4a-0672-11e8-8263-eb22aff63185.png)

![image](https://user-images.githubusercontent.com/12504630/35592763-e44cff80-0672-11e8-842b-92ae6ceab5d7.png)

Windows Smart Screen stopped them both from running so I can't confirm whether they're infected or not but it certainly looks suspicious.

FortiClient picks up neither as being infected, though I haven't run them through a sandbox so can't confirm.
